backend/internal/projects/projects.go

```diff
@@ -97,9 +91,7 @@ func (ps *Projects) Build(ctx context.Context, projectID sdktypes.ProjectID) (sd
 		return nil, err
 	}
 
-	build := kittehs.Must1(sdktypes.BuildFromProto(&sdktypes.BuildPB{
-		ProjectId: projectID.String(),
-	}))
+	build := kittehs.Must1(sdktypes.BuildFromProto(&sdktypes.BuildPB{}))
```

Comment:
Doesn't it make more sense to create a new object directly, without using an empty PB and muting it's impossible translation error?

Reply:
yeah it's silly. going to add an sdktypes.NewBuild() function.